frontend-vue/src/App.vue

```diff
@@ -1,18 +1,23 @@
 <template>
 <div>
-  Joa nichts weiter
-  <br>
+  <h1>Willkommen bei SpotiMann</h1>
   <Login></Login>
-  <SongList></SongList>
+  <SongList v-if="auth.state.loggedIn"></SongList>
```

Comment:
Seh ich das falsch oder prüfen wir auf Client Seite, ob er die Rechte hat, das zu sehen?

Reply:
Das siehst du falsch. Das verhindert nur, dass die Songliste versucht wird anzuzeigen, wenn man nicht eingeloggt ist. Würde man es versuchen, gäbe es eh ein 403.